### PR TITLE
Minor addition to change history

### PR DESCRIPTION
**Pull request by agbeltran**
related to the changes in the basic example

### CODE REVIEW

**Comment by agbeltran** [comment]
Preview here: https://raw.githack.com/w3c/dxwg/agb-change-history-update/dcat/index.html#changes

File: dcat/index.html [hunk 1]
```diff
@@ -6449,6 +6449,9 @@ <h2>Changes since the first public working draft of 17 December 2020</h2>
 
 <p>The document has undergone the following changes since the DCAT 3 first public working draft of 17 December 2020 [[?VOCAB-DCAT-3-20201217]]:</p>
 
+<ul>
+<li><p><a href="#basic-example"></a> has been extended to include tiles, lables and keywords in two different languages (English and Spanish) to illustrate the use of language tags.
```
Comment (andrea-perego) [inline dcat/index.html]:
```suggestion
<li><p><a href="#basic-example"></a> has been extended to include titles, labels, and keywords in two different languages (English and Spanish) to illustrate the use of language tags.
```

File: dcat/index.html [hunk 2]
```diff
@@ -6450,6 +6450,9 @@ <h2>Changes since the first public working draft of 17 December 2020</h2>
 
 <p>The document has undergone the following changes since the DCAT 3 first public working draft of 17 December 2020 [[?VOCAB-DCAT-3-20201217]]:</p>
 
+<ul>
+<li><p><a href="#basic-example"></a> has been extended to include titles, labels, and keywords in two different languages (English and Spanish) to illustrate the use of language tags.
```
Comment (andrea-perego) [inline dcat/index.html]:
```suggestion
<li><p><a href="#basic-example"></a> has been extended to include titles, labels, and keywords in two different languages (English and Spanish) to illustrate the use of language tags.</li>
```

File: dcat/index.html [hunk 3]
```diff
@@ -6450,6 +6450,9 @@ <h2>Changes since the first public working draft of 17 December 2020</h2>
 
 <p>The document has undergone the following changes since the DCAT 3 first public working draft of 17 December 2020 [[?VOCAB-DCAT-3-20201217]]:</p>
 
+<ul>
+<li><p><a href="#basic-example"></a> has been extended to include titles, labels, and keywords in two different languages (English and Spanish) to illustrate the use of language tags.</li>
```
Comment (andrea-perego) [inline dcat/index.html]:
```suggestion
<li><p><a href="#basic-example"></a> has been extended to include titles, labels, and keywords in two different languages (English and Spanish) to illustrate the use of language tags.</p></li>
```

**Review by andrea-perego — APPROVED** [review]

**Review by riccardoAlbertoni — APPROVED** [review]

**Pull request merged**